__test__/index.test.js

```diff
@@ -0,0 +1,17 @@
+const timeout = 15000;
+
+describe('Dummy Test ', () => {
+  beforeAll(async () => {
+    await page.goto('https://www.realdevsquad.com/', {
```

Comment:
Can we use this from constants ?

Reply:
It's just a setup Demo,  we will remove it later. Is Need to replace it?